binrz/rz-test/load.c

```diff
@@ -588,6 +588,22 @@ static RzTestType test_type_for_path(const char *path, bool *load_plugins) {
 	return ret;
 }
 
+static inline bool skip_archos(const char *subname) {
```

Comment:
Can you please add `rz_return_val_if_fail()` for the `subname`? Or check it in the `database_load()` function instead?

Reply:
2c1853b9a575bbd7e432fff3d11ca691cb26cb73 ok added check for `subname` in `skip_archos()`